docs/UserDoc_Chap3_Viewing.rst

```diff
@@ -6,19 +6,19 @@ This section will guide users and visitors to the site on viewing metadata and p
 Viewing metadata records
 ------------------------
 
-Clicking on the title of a record (from either the search results or those presented on the main page) will allow the user to view the metadata 
-record. Below the record title, the **abstract** provides a brief description of what the dataset (or service) is and what it is used for. The 
-spatial **extent** of the resource can be seen in the map below, in the **Discover data** section. Access to the data itself can be gained from the options listed in the 
+Clicking on the title of a record (from either the search results or those presented on the main page) will allow the user to view the metadata
+record. Below the record title, the **abstract** provides a brief description of what the dataset (or service) is and what it is used for. The
+spatial **extent** of the resource can be seen in the map below, in the **Discover data** section. Access to the data itself can be gained from the options listed in the
 **Links** section.
 
 |userdoc_fig_3_1_1_DefaultViewA|
 
 **Figure 3.1.1:** Default record view
 
-Scrolling down the page, the **Use contraints** section provides information on licencing and the **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats are provided. Information on the history of a resource and how 
+Scrolling down the page, the **Use contraints** section provides information on licencing and the **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats are provided. Information on the history of a resource and how

```

Comment:
contraints -> constraints

Reply:
Scrolling down the page, the **Use constraints** section provides information on licensing.  The **Technical information** headings provide details on topic categories and keywords, maintenance frequency, coordinate systems and data formats.

Reply:
updated